### ISSUE TITLE
Remove F* in flake 

### ISSUE THREAD

**Issue by msprotz** [AeneasVerif/eurydice#88]
Until https://github.com/FStarLang/karamel/issues/496 is fixed

**Comment by msprotz** [comment]
@Nadrieril how can I instruct Nix to run only "make minimal" in karamel, so that the build does not require an F* dependency?

**Comment by msprotz** [comment]
Yeah if that's ok it would be great -- you can probably do it much faster than I can. Basically if you invoke only "make minimal" in krml it builds without F*.

**Comment by Nadrieril** [comment]
Karamel itself uses `FSTAR_HOME="dummy"` in their flake ([here](https://github.com/FStarLang/karamel/blob/5f7e9be838ba8ec0ed3323132e6beb6276f6acf2/.nix/karamel.nix#L63)), we could just do that too?

**Comment by msprotz** [comment]
yes that would be fine -- thanks!!
